docs/testing/debug-with-vs-extension.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Office Addin Debugger Extension for Visual Studio Code
+description: 'Use the Visual Studio Code extension Office Addin Debugger to debug your Office add-in.'
```

Comment:
```suggestion
description: 'Use the Visual Studio Code extension Office Addin Debugger to debug your Office Add-in.'
```
Also, we need to escalate the name of the debugger... "Office Addin" with no hyphen is inconsistent with our branding, as you know. Can you ask Louis why they did it this way?